service/surf/apps/materials/views.py

```diff
@@ -550,3 +550,20 @@ def add_share_counters_to_materials(materials):
         m["sharing_counters"] = SharedResourceCounterSerializer(many=True).to_representation(qs.all())
 
     return materials
+
+
+class MaterialSetAPIView(APIView):
+    def get(self, request, *args, **kwargs):
+        serializer = MaterialShortSerializer(data=request.GET)
+        serializer.is_valid(raise_exception=True)
+        data = serializer.validated_data
+        results = _get_material_by_external_id(request, data['external_id'])
+        parts = results[0]['has_part']
+
+        materials = []
+        for part in parts:
+            material = _get_material_by_external_id(request, part)[0]
```

Comment:
Dit lijkt mij vrij inefficient, is het niet mogelijk om meerdere materialen in 1 keer te query'en?

Reply:
done